web/hooks/github.go

```diff
@@ -47,11 +47,31 @@ func (wh GitHubWebHook) Handle(w http.ResponseWriter, r *http.Request) {
 	case *github.PushEvent:
 		wh.logger.Debug(log.IndentJson(e))
 		wh.handlePush(e)
+	case *github.InstallationEvent:
+		wh.logger.Debug(log.IndentJson(e))
+		wh.handleInstallation(e)
 	default:
 		wh.logger.Debugf("Ignored event type %s", github.WebHookType(r))
 	}
 }
 
+func (wh GitHubWebHook) handleInstallation(payload *github.InstallationEvent) {
```

Comment:
Could you add a brief explainer for when this will be called. Looks like there is one app per course. Things like this would be helpful to draw a mental picture for reads of the code.

Reply:
Actually, we don't need to listen for an installation event when creating a course, I'll remove this method when the alternative is finished.